core/src/main/resources/org/broadinstitute/dsde/rawls/liquibase/changesets/20250211_add_workspace_spend_report_table.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="dummy" xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+    <changeSet logicalFilePath="dummy" author="sehsan" id="create_WORKSPACE_SPEND_REPORT_TABLE">
+        <createTable tableName="WORKSPACE_SPEND_REPORT">
+            <column name="GOOGLE_PROJECT_ID" type="VARCHAR(254)">
+                <constraints nullable="false" primaryKey="true" primaryKeyName="PK_WORKSPACE_SPEND_REPORT"/>
+            </column>
+            <column name="REPORT_START_DATE" type="DATETIME">
+                <constraints nullable="false" primaryKey="true" primaryKeyName="PK_WORKSPACE_SPEND_REPORT"/>
+            </column>
+            <column name="REPORT_END_DATE" type="DATETIME">
+                <constraints nullable="false" primaryKey="true" primaryKeyName="PK_WORKSPACE_SPEND_REPORT"/>
+            </column>
+            <column name="TOTAL_SPEND" type="NUMBER(10,2)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="TOTAL_COMPUTE" type="NUMBER(10,2)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="TOTAL_STORAGE" type="NUMBER(10,2)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="OTHER_SPEND" type="NUMBER(10,2)">
+                <constraints nullable="false"/>
+            </column>
```

Comment:
Does `TOTAL_SPEND` need its own column? Is that always going to be the sum of TOTAL_COMPUTE+ TOTAL_STORAGE+ OTHER_SPEND ?

Reply:
Ah I think I got confused with the inconsistency we found in the UI but I forgot that we just don't show the other spend category. What do you think about making it a computed column?

Reply:
Yeah it's just the sum of the other categories, which the query [calculates and returns](https://github.com/broadinstitute/rawls/blob/88bcc22220beeb9aa328d09c46cc660c10ebdb9d/core/src/main/scala/org/broadinstitute/dsde/rawls/spendreporting/SpendReportingService.scala#L403) for you

Reply:
Yeah, I just decided to delete it